specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/chambers.json

```diff
@@ -0,0 +1,1042 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Delete Chamber": {
+            "$ref": "./examples/ChambersDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.ModSimWorkbench/workbenches/{workbenchName}/chambers": {
+      "get": {
+        "tags": [
+          "chambers_List"
+        ],
+        "operationId": "chambers_List",
+        "description": "Lists all Chambers.",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroup",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "workbenchName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the workbench."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request is successful. The operation returns the list of all Chambers",
+            "schema": {
+              "$ref": "#/definitions/ChamberPageableList"
+            }
+          },
+          "default": {
+            "description": "",
+            "schema": {
+              "$ref": "./modsimworkbenches.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all Chambers": {
+            "$ref": "./examples/ChambersList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ChamberPageableList": {
+      "description": "Microsoft.Sdw.Core.ResourceMetadata.PageableList implementation specific to pageable lists of Microsoft.Sdw.Core.ResourceMetadata.AzureResource`1.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "Gets or sets the URL (link) to the next result set.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Gets or sets the values to emit from this list.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Chamber"
+          },
+          "x-ms-identifiers": [
+            "value"
+          ]
+        }
+      }
+    },
+    "FileUploadParameters": {
+      "description": "File Upload Parameters DTO.",
```

Comment:
> "File Upload Parameters DTO.", [](http://example.com/codeflow?start=20&length=31)

Can you expand or eliminate the DTO acronym?

Reply:
removed